services/storage/tests/unit/test_async_jobs.py

```diff
@@ -64,30 +49,40 @@ class _MockCeleryClient:
     abort_task_object: Exception | None = None
 
     async def send_task(self, *args, **kwargs) -> TaskUUID:
+        _ = args
+        _ = kwargs
         assert self.send_task_object is not None
         if isinstance(self.send_task_object, Exception):
```

Comment:
why is this MockClient still around? I thought we said all these mocks go away?

Reply:
remove all mocks form this module